ddtrace/tracer/span_test.go

```diff
@@ -204,6 +204,31 @@ func TestSpanSetTag(t *testing.T) {
 	assert.Equal("false", span.Meta["some.other.bool"])
 }
 
+func TestSpanSetTagError(t *testing.T) {
+	assert := assert.New(t)
+
+	span1 := newBasicSpan("web.request")
+	span1.setTagError(errors.New("error value with no trace"), &errorConfig{noDebugStack: true})
+	assert.Equal(int32(1), span1.Error)
+	assert.Equal("error value with no trace", span1.Meta[ext.ErrorMsg])
+	assert.Equal("*errors.errorString", span1.Meta[ext.ErrorType])
```

Comment:
I believe these things are already covered in other tests, so no need to reiterate them.